cmd/testbeacon.go

```diff
@@ -0,0 +1,124 @@
+// Copyright © 2022-2024 Obol Labs Inc. Licensed under the terms of a Business Source License 1.1
+
+package cmd
+
+import (
+	"context"
+	"io"
+	"sort"
+	"time"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/exp/maps"
+
+	"github.com/obolnetwork/charon/app/errors"
+)
+
+type testBeaconConfig struct {
+	testConfig
+	Endpoints []string
+}
+
+func newTestBeaconCmd(runFunc func(context.Context, io.Writer, testBeaconConfig) error) *cobra.Command {
+	var config testBeaconConfig
+
+	cmd := &cobra.Command{
+		Use:   "beacon",
+		Short: "Run multiple tests towards beacon nodes",
+		Long:  `Run multiple tests towards beacon nodes. Verify if the current setup is suitable for mainnet cluster.`,
```

Comment:
Too vague `Verify if the current setup is suitable for mainnet cluster.`, what if we say "Verify that Charon can interact with Beacon Node."?

Reply:
Agree, added `efficiently` as well